modules/agent-configuration-parameters.adoc

```diff
@@ -127,4 +127,12 @@ For more information, see "Root device hints" in the "Setting up the environment
 |The host network definition.
 The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
 |A dictionary of host network configuration objects.
+
+|minimalISO:
```

Comment:
Do I need to specify that if platform is external, there's another method you'd have to use to generate a minimal ISO?

Reply:
If platform is External then a minimalISO will always be generated, regardless of this setting.

Reply:
I believe we should also include something about the Minimal ISO in the bootArtifactsBaseURL description, as it currently seems more specific to PXE. What do you think?